Collection Page: convert html to markdown in description field

ekraffmiller opened the pull request:
## What this PR does / why we need it:
The Get Collection use case in js-dataverse was updated to return the html in the description field, rather than markdown.  This PR converts the html returned from js-dataverse into markdown for the Collection Page. (In Edit Collection Page, it will show the original html).

## Which issue(s) this PR closes:

- Closes #599 

## Special notes for your reviewer:
This change depends on a js-dataverse PR https://github.com/IQSS/dataverse-client-javascript/pull/265.  When that PR is approved, package.json should be updated to use the alpha version of the library.

## Suggestions on how to test this:

Step 1: Run the Development Environment

Execute npm i.
Navigate with cd packages/design-system && npm i && npm run build.
Return with cd ../../.
Ensure you have a .env file similar to .env.example, with the variable VITE_DATAVERSE_BACKEND_URL=http://localhost:8000.
Navigate with cd dev-env.
Start the environment using ./run-env.sh unstable.
To verify the environment, visit http://localhost:8000/ and check your local Dataverse installation.

Step 2: Test the feature

Create a Collection and add html in the description field.  The Collection Page should show the rendered html.  Go to Edit Collection Page. It should show the original html that was submitted.

g-saracca reviewed (changes requested):
All good! Just 2 minor comments

g-saracca reviewed (dismissed):
Approving, thanks!

ofahimIQSS commented:
test/e2e is failing on this - please advise :)

ekraffmiller commented:
@ofahimIQSS I think this is a timing issue - the csv file has be ingested and converted to a tab-delimited file, which I think happens asynchronously on the server side.  (I ran it locally and can't reproduce the error.)I updated the test to check the filterBy using text files, which don't have to be ingested.
@g-saracca, can you review my change?

ofahimIQSS commented:
testing passed but need confirmation on why the UI review and tests are pending

g-saracca commented:
@ofahimIQSS  All UI Reviews passing.
